tests/unit/s2n_psk_key_exchange_modes_extension_test.c

```diff
@@ -0,0 +1,142 @@
+/*
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * A copy of the License is located at
+ *
+ *  http://aws.amazon.com/apache2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+#include "s2n_test.h"
+#include "testlib/s2n_testlib.h"
+
+#include "tls/s2n_psk.h"
+
+/* Include source to test static functions */
+#include "tls/extensions/s2n_psk_key_exchange_modes.c"
+
+int main(int argc, char **argv)
+{
+    BEGIN_TEST();
+
+    /* Test: s2n_psk_key_exchange_modes_send */
+    {
```

Comment:
The `psk_key_exchange_modes` extension MUST be  sent by the client and received by the server (and not vice versa), can we add a unit test to verify it?

Reply:
I'm not sure that's necessary. That should be covered by the extension framework, which is tested separately.